DigitalOcean.API/Models/Responses/HealthCheck.cs

```diff
@@ -35,13 +32,11 @@ public class HealthCheck {
 		/// <summary>
 		/// The number of times a health check must fail for a backend Droplet to be marked "unhealthy" and be removed from the pool.
 		/// </summary>
-		[JsonProperty("unhealthy_threshold")]
 		public int UnhealthyThreshold { get; set; }
 
 		/// <summary>
 		/// The number of times a health check must pass for a backend Droplet to be marked "healthy" and be re-added to the pool.
 		/// </summary>
-		[JsonProperty("healthy_threshold")]
 		public int HealthyThreshold { get; set; }
```

Comment:
Since we have an explicit Requests class now, remove JsonPropertyAttributes since they are always used for requesets not responses.